toolkit/imageconfigs/scripts/cleanup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -eux
+
+# cleanup symlinks created by the toolkit that are not needed for base images
+if [ -L /srv ]; then
+  echo "Removing /srv symlink"
+  rm /srv
+else
+  echo "/srv symlink does not exist"
+fi
+
+# cleanup any logs that may have been created during the build
+if [ -d /var/log ]; then
+  echo "Clearing /var/log"
+  rm -rf /var/log/*
```

Comment:
Out of interest, what files are getting added to `/var/log` during the build?

Reply:
Nothing specific that I'm aware of. Trying to align to https://github.com/microsoft/CBL-Mariner/blob/main/toolkit/imageconfigs/postinstallscripts/core-container/cleanup.sh where it makes sense.